CHANGELOG.md

```diff
@@ -4,4 +4,13 @@ All notable changes to the "vscode-servicefabric" extension will be documented i
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
 ## [Unreleased]
-- Initial release
```

Comment:
Add these in the reverse order, with latest changelog at the top.

Reply:
Thanks Anant, made the change.